_posts/elements/2015-03-25-example-headings-new.md

```diff
@@ -0,0 +1,50 @@
+---
+layout:       element
+title:        Example - Demo Headings
+date:         2015-02-25 00:00:00
+
+categories:   elements
+tags:
+- example
+
+slug:         example-headings-new
+scssurl:      "https://github.com/edx/ux-pattern-library/blob/master/[path_to_source_sass_file].scss"
+
+description:  Examples of headings in a semi-real world context.
+
+info:  Vestibulum id ligula porta felis euismod semper. Donec id elit non mi porta gravida at eget metus. Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Etiam porta sem malesuada magna mollis euismod.
+---
+
+<section class="group">
+    <article>
+        <h1 class="hd-1">Proident laborum ut nulla consequat kevin adipisicing dolar.</h1>
+        <p>In dolor filet mignon est. Flank tail eu, pork belly sed incididunt shank short ribs id pig sausage meatball. Ribeye landjaeger pancetta pork belly shank officia incididunt in dolor excepteur swine ham. In ut short ribs consectetur hamburger ipsum.</p>
```

Comment:
Please reduce this to just one paragraph of text (its good to see how a headings is rendered against copy, but I don't think we need so much).